packages/ifpack2/src/Ifpack2_Details_Chebyshev_def.hpp

```diff
@@ -631,6 +635,8 @@ setParameters (Teuchos::ParameterList& plist)
     "\" parameter (also called \"eigen-analysis: iterations\") must be a "
     "nonnegative integer.  You gave a value of " << eigMaxIters << ".");
 
+  tol = plist.get ("chebyshev: eigenvalue tol", tol);
```

Comment:
So far, we don't have other eigensolvers in Chebyshev. But even if we use Lanczos, it does allow to use tolerance for stopping. So I think it could be used for both.